src/Containers/CreateEvent/CreateEvent.js

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from 'react';
 import styled from 'styled-components';
 import Header from '../../Components/Header/Header.js';
+import { NavLink } from 'react-router-dom';
```

Comment:
Does this add special styling specifically when we create an event?

Reply:
Yeah, the style-components is a package to let you modify the CSS in the component file. Collie showed me how to do it!